plugin/http.go

```diff
@@ -108,6 +103,16 @@ func NewHTTP(log *util.Logger, method, uri string, insecure bool, cache time.Dur
 		Transport: p.Client.Transport,
 	}
 
+	if cache > 0 {
+		cacheHeader := fmt.Sprintf("max-age=%d, must-revalidate", int(cache.Seconds()))
+		p.Client.Transport = &transport.Decorator{
+			Decorator: transport.DecorateHeaders(map[string]string{
```

Comment:
Should we only do this if cache-control is not present to allow the device to stay in control (send etag, whatever)?

Reply:
But why then use the cache parameter? If the device sends not useful cache control header (and auch device will show up sooner or later) it is useful to override the header

Reply:
Assumption: if header is set its meaningful

Reply:
If you can expect the device to send useful Cache Control headers, just dont set cache: or set it to 0 and it won't interfere. I think that is reasonable.